Put 5 html templates in respective folders, made quizSummary folder

samirachamplin opened the pull request:
First 5 (alphabetically) redone html templates

Commit:
Put 5 html templates in respective folders, made quizSummary folder

kincaidoneil commented:
Hey! The HTML itself looks really good. Just a few logistical things:
1. It'd be best if you could copy-paste just the HTML in the within the `<body>` portion (but not the `<body>` tags themselves) into the template files that Leo/Daniel committed (sorry these weren't there earlier! You may need to sync Github). The doctype and stylesheets and all that jazz we'll just have in a single `index.html` file, so Angular can do it's magic swapping the templates in and out.
2. Make sure you're using _tabs_ to indent and not spaces (for code consistency)
3. For now, only one question page is fine. We'll do one of two things on Thursday: either put some Angular data bindings into your question HTML template so it dynamically creates a `<div>` for each question according to a JSON file (in other words, it'd change itself based on however many questions are necessary), or we'll postpone that to next week and just choose an arbitrary number of questions to a page (I'm not sure what we'll be able to get done for the alpha).
4. Take a brief look at the `app.js` file in the app folder. For each of the different templates/pages, there's the name of a route for each (kinda like in Sean's Angular demo app). And then, in each of the templates, a link to another template is preceded with a "#" and then the name of the route. For example, the link to the choose quiz page would be "#/chooseQuiz" (that'd be the `href` attribute of an `<a>` element). It'd be awesome if you could update those! (again, Leo/Daniel just added them)
5. Lastly is data binding (I'm thinking we'll start this on class in Thursday, probably a collaboration between the HTML group and the Angular group). Basically, we need to take the data from April and Max's JSON file and using Angular's special syntax to map it to the templates. I guess what I'm saying is (if and only if you have time on top of the rest of my suggestions!) take a brief look specifically at the data binding portion of an Angular tutorial so you have a good idea what's going on for Thursday.

Other than that, all the HTML looks solid :smiley:

Commit:
Copied HTML to respective files, added quit page folder

Need to figure out multiple question page templates for different
number of answers

Commit:
Template edits

Removed stuff outside body and fixed spacing

Commit:
Merge branch 'master' into final-html-templates

Commit:
Put quiz description template in correct place

lagerasimov17 committed:
quit page folder and file additions (HTML and JS)

lagerasimov17 committed:
update consistency in names

Commit:
Removed body tags

Commit:
Merge branch 'master' into final-html-templates

lagerasimov17 committed:
routing for quit page

Commit:
Fixed route in privacy policy page

lagerasimov17 committed:
routing for quiz description

allenw16 committed:
Added HTML
